examples/src/example_simplejs.ts

```diff
@@ -22,7 +22,7 @@ LE.marker([38.912753, -77.032194])
     .addTo(map)
     .openPopup();
 
-const harpGL = new LE.HarpGL({
+const harpGL = new LE.harp.HarpGL({
```

Comment:
changing LE.HarpLGL to LE.harp.HarpGL doesn't look nice, this is public API and we should avoid t change it too often. Isn't there a way to support `LE.HarpGL`?